src/modules/claim/claim.service.ts

```diff
@@ -112,12 +112,16 @@ export class ClaimService {
     rq: IClaimIssuance,
   ): Promise<ClaimHandleResult> {
     const claim: RoleClaim = await this.getById(rq.id);
+    const {
+      claimData: { claimType, claimTypeVersion },
+    } = jwt.decode(rq.issuedToken) as DecodedClaimToken;
```

Comment:
@Harasz This needs to be moved into first `if`. See https://github.com/energywebfoundation/iam-cache-server/pull/221/files